app/utils/bundle-importer.js

```diff
@@ -70,11 +70,34 @@ YUI.add('bundle-importer', function(Y) {
     },
 
     /**
-      Fetch the dry-run output from the Deployer.
+      Fetch the dry-run output from the Guiserver.
 
       @method fetchDryRun
+      @param {String} bundleYAML The bundle file contents.
     */
-    fetchDryRun: function() {},
+    fetchDryRun: function(bundleYAML) {
+      this.env.getChangeSet(bundleYAML, this._handleFetchDryRun.bind(this));
+    },
+
+    /**
+      Handles the dry run response.
+
+      @method _handleFetchDryRun
+      @param {Object} response The processed response data.
+    */
+    _handleFetchDryRun: function(response) {
+      if (response.err) {
```

Comment:
Those are the bundle validation errors, and they are assumed to be human readable and friendly.
I wonder if we should show them in the notification directly, formatting them so that they are displayed one per line.
Since this is an array of messages, maybe a better name would be response.errors.